style.css

```diff
@@ -199,6 +199,322 @@ body {
   transition: .3s ease;
 }
 
+/*Вторая страница*/
+
+.second-page {
+  background-color: #fae8e4;
+  min-height: 980px;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.second-page__img {
+  margin-right: 135px;
+  flex: 3;
+}
+
+.second__img_purses {
+  width: 90%;
+  padding-left: 147px;
+  padding-top: 115px;
+}
+
+.second__img_front-purse {
+  display: none;
+}
+
+.second__wrap {
+  margin: 55px 64px 0 0;
+  flex: 2;
+}
+
+.second-page__article {
+  width: 400px;
+  background-color: #fff;
+  padding: 42px 72px 47px 60px;
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 45px;
+  box-shadow: 20px 40px 40px -48px;
+}
+
+.second-page__title {
+  font: 700 45px 'Montserrat-Bold', sans-serif;
+  letter-spacing: -1px;
+  margin-bottom: 11px;
+}
+
+.second-page__subtitle {
+  font: 600 16px 'Montserrat-Bold', sans-serif;
+  letter-spacing: 2.3px;
+  margin-bottom: 67px;
+  opacity: .5;
+}
+
+.second-page__description {
+  font: 600 16px/23px 'Montserrat-Regular', sans-serif;
+  letter-spacing: .1px;
+  margin-bottom: 45px;
+  opacity: .3;
+}
+
+.second__options {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+}
+
+.second-page__color-options {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 55px;
+}
+
+.eclipse {
+  width: 35px;
+  height: 35px;
+  border-radius: 50%;
+  cursor: pointer;
+  margin-right: 14px;
+}
+
+.eclipse-white {
+  background-color: #dedede;
+}
+
+.eclipse-pink {
+  transform: scale(1.2);
+  background-color: #f29f9f;
+}
+
+.eclipse-blue {
+  background-color: #a8bff9;
+}
+
+.eclipse-green {
+  background-color: #90bd8f;
+}
+
+.eclipse-white:hover, 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Flack_project/pullRequest?prid=6358146)